WebView/app/src/main/assets/weather.json

```diff
@@ -0,0 +1,6 @@
+{
+  "class": "weather",
+  "location": "WebView City",
+  "description" : "Thunderstorms and 65",
+  "icon": "https://gcoleman799.github.io/res/drawable/thunderstorm.png"
```

Comment:
Possibly for a follow-up PR: I think the icon should be determined in the client (based on the weather description), rather than specified in the JSON data.